lib/render.js

```diff
@@ -43,3 +48,37 @@ exports.markdownEngine = (nunjucksEnv) => {
     })
   }
 }
+
+// Use with app.engine('.njk', nunjucksAndMarkdownEngine(nunjucksEnv))
+// Express can't handle with an engine with multiple file extensions
+exports.nunjucksAndMarkdownEngine = function (nunjucksEnv) {
+  return (filePath, options, callback) => {
```

Comment:
not a strong opinion but might it be easier to read as 3 functions? To do
 - matter
 - nunjucks
 - markdown

Reply:
Yeah, I tried that initially, but I found it was harder to follow the logic.